client/blocks/inline-help/inline-help-forum-guide.jsx

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import { preventWidows } from 'lib/formatting';
+
+const FORUM_LINK = '//en.forums.wordpress.com';
+
+const InlineHelpRichForumGuide = ( { translate = noop } ) => (
+	<div className="inline-help__forum-view">
+		<p>
+			{ preventWidows(
+				translate(
+					'Post a new question in our {{strong}}public forums{{/strong}}, ' +
```

Comment:
Copy change suggestion:
We could break it up in two parts to make it easier to comprehend. And maybe also change the `may` to a more promising `will`.
> "In our public forums, you can look for existing answers to your question. Alternatively, you can post a new question that will be answered by our helpful community members."